File: LeetCode-Problems/Easy/Jewels-and-Stones.py
# Python
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        count = 0;
        for x in range(0, len(stones)):
            for y in range(0, len(jewels)):
                if stones[x:x+1] == jewels[y:y+1]:
                    count += 1
        return count
        

File: LeetCode-Problems/Easy/Jewels-and-Stones.java
//Java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int count = 0;
        for (int i = 0; i < stones.length(); i++){
            for (int j = 0; j < jewels.length(); j++){
                if (jewels.substring(j, j+1).equals(stones.substring(i, i+1))){
                    count++;
                }
            }
        }
        return count;
    }
}
